test/5hw/home work_11/1_exrcise.py

```diff
@@ -0,0 +1,54 @@
+class Company:
+    def __init__(self, name, industry, headquarters, founder, year_founded):
+        self.name = name
+        self.industry = industry
+        self.headquarters = headquarters
+        self.founder = founder
+        self.year_founded = year_founded
+
+    def get_name(self):
+        return self.name
+
+    def get_industry(self):
+        return self.industry
+
+    def get_headquarters(self):
+        return self.headquarters
+
+    def get_founder(self):
+        return self.founder
+
+    def get_year_founded(self):
+        return self.year_founded
+
+    def set_name(self, name):
+        self.name = name
+
+    def set_industry(self, industry):
+        self.industry = industry
+
+    def set_headquarters(self, headquarters):
+        self.headquarters = headquarters
+
+    def set_founder(self, founder):
+        self.founder = founder
+
+    def set_year_founded(self, year_founded):
+        self.year_founded = year_founded
```

Comment:
we do not write get_ set_ in Python. We use @property.


Also, where are all the other functions? static classmethod, functions of a class?